### PR TITLE
Application insight with logging workspace 

### PR DESCRIPTION
**Pull request by kristianschneider**
This implements :
https://app.zenhub.com/workspaces/mighty-ducks---the-outlaws-6193fe815d79fc0011e741b1/issues/energinet-datahub/the-outlaws/166

which regarding the new approach for application insight being associated with a logging workspace

### CODE REVIEW

File: azure/application-insights/main.tf [hunk 1]
```diff
@@ -33,4 +34,12 @@ resource "azurerm_application_insights" "this" {
       tags,
     ]
   }
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "log-${lower(var.name)}-${lower(var.project_name)}-${lower(var.environment_short)}-${lower(var.environment_instance)}"
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  sku                 = "PerGB2018"
```
Comment (djorgensendk) [inline azure/application-insights/main.tf:43]: Will "PerGB2018" always be the right choice of sku, or should this be configurable through a variable with "PerGB2018" as the default value?

File: azure/application-insights/variables.tf [hunk 2]
```diff
@@ -41,6 +41,11 @@ variable location {
   description = "(Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created."
 }
 
+variable retention_in_days{
+  type        = string
+  description = "(Required) How long are logs retained."
+}
+
```
Comment (djorgensendk) [inline azure/application-insights/variables.tf:48]: How about assigning a default value, which has been agreed with the architects? I'd think Martin would have an opinion about this

File: azure/application-insights/main.tf (hunk 1 above)
Comment (kristianschneider) [reply azure/application-insights/main.tf:43]: PerGB2018 is the only sku right now. You typically adjust by setting retention period

File: azure/application-insights/variables.tf (hunk 2 above)
Comment (kristianschneider) [reply azure/application-insights/variables.tf:48]: God idea. Ill run it past him

**Comment by Renetnielsen** [comment]
Closing this since it was added through @45

**Pull request closed without merging**